src/MovieModal/MovieModal.js

```diff
@@ -5,12 +5,11 @@ import './MovieModal.css';
 class MovieModal extends React.Component {
   constructor(props) {
     super(props);
-    console.log(this.props)
     this.state = {
```

Comment:
Because all of this info is coming through as props, does this component NEED to be a class-based component, or can it be a function component?